device_coupler/config/daq_config.yaml

```diff
@@ -20,7 +20,7 @@ run_trigger:
   vlan_start: 200
   vlan_end: 205
   auto_session: True
-  docker_bridge_ip: 172.17.0.1
+  grpc_target_ip: 172.17.0.1
```

Comment:
not grpc -- not a useful prefix. Think of the semantics of what it's doing and come up with a descriptive name.  It's the IP address for some machine -- what is the purpose of that target?